nnpdf_data/nnpdf_data/filter_utils/athena_utils.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+from pathlib import Path
+from typing import Optional, Union
+
+import yaml
+import numpy as np
+
+
+def read_excel(path_xlsx: Path, beams: tuple) -> pd.DataFrame:
+    """Parse the xlsx file containing all the information regarding
+    the projections and returns the ones corresponding to the chosen
+    beam energies.
+
+    Parameters
+    ----------
+    path_xlsx : Path
+        path to the xlsx file
+    beams: tuple
+        tuple specifying the beam energies of the lepton & proton
+
+    Returns
+    -------
+    pd.DataFrame
+        returns a panda table corresponding to the chosen beams
+    """
+    xdf = pd.read_excel(path_xlsx)
+    el, ep = beams
+    df_beam = xdf[(xdf["El"] == el) & (xdf["Eh"] == ep)]
+    return df_beam[df_beam["y"] <= 1]
+
+
+def fluctuate_data(central: np.ndarray, abserr: np.ndarray) -> np.ndarray:
```

Comment:
Why is the data fluctuated?

Reply:
These are not actually real experimental data but rather pseudodata, so the central values are defined as fluctuations of some theory predictions provided with the projection uncertainties.

Reply:
Yes, this is indeed intended because we do not have predictions for this energy yet.

Reply:
Ah right, I should've looked at the metadata.yaml and I could've found that out. Perhaps you can add to the docstring of the function that this is done because these datasets are pseudodata

Reply:
I add some extra texts in the docstring to reflect this.